src/signals/settings/categories/components/CategoryForm.tsx

```diff
@@ -55,87 +53,123 @@ export interface Props {
 
 export const CategoryForm = ({
   formMethods,
-  formValues,
   history,
   onCancel,
   onSubmit,
   readOnly,
   responsibleDepartments,
   isMainCategory,
   isPublicAccessibleLabel,
-}: Props) => (
-  <FormProvider {...formMethods}>
-    <FormContainer>
-      <Form onSubmit={onSubmit} data-testid="detail-category-form">
-        <Row>
-          <StyledColumn
-            span={{ small: 1, medium: 2, big: 4, large: 5, xLarge: 5 }}
-          >
-            <div>
-              <FieldGroup>
-                {!isMainCategory ? (
-                  <Input
-                    {...formMethods.register('name')}
-                    disabled={readOnly}
-                    hint="Het wijzigen van de naam heeft geen invloed op het type melding"
-                    id="name"
-                    label="Naam"
-                    name="name"
-                    readOnly={readOnly}
-                    type="text"
+}: Props) => {
+  const watchIs_Active = useWatch({
+    name: 'is_active',
+    control: formMethods.control,
+  })
+
+  const watchIs_Public_Accessible = useWatch({
+    name: 'is_public_accessible',
+    control: formMethods.control,
+  })
+
+  return (
+    <FormProvider {...formMethods}>
+      <FormContainer>
+        <Form onSubmit={onSubmit} data-testid="detail-category-form">
+          <Row>
+            <StyledColumn
+              span={{ small: 1, medium: 2, big: 4, large: 5, xLarge: 5 }}
+            >
+              <div>
+                <FieldGroup>
+                  {!isMainCategory ? (
+                    <Input
+                      {...formMethods.register('name')}
+                      disabled={readOnly}
+                      hint="Het wijzigen van de naam heeft geen invloed op het type melding"
+                      id="name"
+                      label="Naam"
+                      name="name"
+                      readOnly={readOnly}
+                      type="text"
+                    />
+                  ) : (
+                    <>
+                      <StyledDefinitionTerm>
+                        <strong>Naam</strong>
+                      </StyledDefinitionTerm>
+                      <dd data-testid="name">{formMethods.getValues().name}</dd>
```

Comment:
formMethods.getValues('name') is more concise. Believe it should work

Reply:
all right, changed it.